src/wrapper.rs

```diff
@@ -506,51 +550,46 @@ impl Dxc {
         Ok(Self { dxc_lib })
     }
 
-    pub(crate) fn get_dxc_create_instance(&self) -> Result<Symbol<DxcCreateInstanceProc>> {
```

Comment:
Where does this generic type come from?

Reply:
It is defined in `ffi.rs`, and makes it a bit easier to extract a COM object of a specific type windows-rs style, instead of having a `c_void` and bunch of casts here.